Salvi6God/bounty_hunter_x
# adds planeTypes.js

@martensonbj, @joshuajhun, @rrgayhart, @thompickett 

What's this PR do?
Begins an overhaul of the way we generate enemy planes.

Where should the reviewer start?
Start within the enemyLevels file and follow each of the three case statements.

How should this be manually tested?
Check that all the tests work, then start the game and see that the plane sprites show up.

Any background context you want to provide?
There is a grand amount of repetitive code in the way in which the enemy planes get generated/re-generated. Breaking out the plane objects to a new file helps define what code is being repeated in both files.

What gif best describes this PR or how it makes you feel?
http://i.imgur.com/FG2wcV7.gif
